lib/rules/generator-star-spacing.js

```diff
@@ -43,20 +43,27 @@ module.exports = function(context) {
     }
 
     /**
+     * Checks whether the current node is a class or object method.
+     * @returns {boolean} True if the current node is a method.
+     */
+    function isMethod() {
+        var parent = context.getAncestors().pop();
```

Comment:
Probably better to pass in the node and use `node.parent` to make sure you have the right data.